nsv13/code/modules/overmap/overmap.dm

```diff
@@ -62,61 +74,85 @@
 	var/bump_impulse = 0.6
 	var/bounce_factor = 0.2 // how much of our velocity to keep on collision
 	var/lateral_bounce_factor = 0.95 // mostly there to slow you down when you drive (pilot?) down a 2x2 corridor
+
 	var/brakes = FALSE //Helps you stop the ship
-	var/fire_delay = 5
-	var/weapon_range = 10 //Range changes based on what weapon youre using.
-	var/obj/railgun_overlay/railgun_overlay
-	var/atom/last_target //Last thing we shot at, used to point the railgun at an enemy.
 	var/rcs_mode = FALSE //stops you from swivelling on mouse move
-	var/fire_mode = FIRE_MODE_PDC //What gun do we want to fire? Defaults to railgun, with PDCs there for flak
 	var/move_by_mouse = TRUE //It's way easier this way, but people can choose.
-	var/faction = null //Used for target acquisition by AIs
-	var/sprite_size = 64 //Pixels. This represents 64x64 and allows for the bullets that you fire to align properly.
-	var/torpedoes = 15 //Prevent infinite torp spam
-	var/pdc_miss_chance = 20 //In %, how often do PDCs fire inaccurately when aiming at missiles. This is ignored for ships as theyre bigger targets.
-	var/list/dent_decals = list() //Ships get visibly damaged as they get shot
-	var/damage_states = FALSE //Did you sprite damage states for this ship? If yes, set this to true
-	var/list/torpedoes_to_target = list() //Torpedoes that have been fired explicitly at us, and that the PDCs need to worry about.
-	var/main_overmap = FALSE //There can only be one of these per game! This denotes that this ship is the "hero ship" and what the players fly. This links it to all the station areas by default
-	var/area_type = null //Set the type of the desired area you want a ship to link to, assuming it's not the main player ship.
-	var/list/linked_areas = list() //List of all areas that we control
-	var/list/mobs_in_ship = list() //A list of mobs which is inside the ship. This is generated by our areas.dm file as they enter / exit areas
-	var/impact_sound_cooldown = FALSE //Avoids infinite spamming of the ship taking damage.
+
+	// Mobs
+	var/mob/living/pilot //Physical mob that's piloting us. Cameras come later
 	var/mob/living/gunner //The person who fires the guns.
 	var/list/operators = list() //Everyone who needs their client updating when we move.
+	var/list/mobs_in_ship = list() //A list of mobs which is inside the ship. This is generated by our areas.dm file as they enter / exit areas
```

Comment:
So this means that ALL ships get a empty list JUST for this? Thats just fine you dont need to do anything about this. A better method which reduces memory usage will be worked out later on. Probably a larger rework of the overmap paths in general.